src/commands/dep/cargo/install.ts

```diff
@@ -68,37 +65,31 @@ export default class Install extends BaseCommand<typeof Install> {
   };
 
   async run(): Promise<void> {
-    const { args, flags, maybeFluenceConfig } = await initCli(
+    const { args, flags, fluenceConfig } = await initCli(
```

Comment:
I like this naming much more :)

Reply:
What about the renaming in other commands?

Reply:
it's currently if you pass "true" the command will make sure you have fluence project. If you don't pass that - it will possibly be null. So it's not a renaming, I just changed the requirements for this command, because now global dependency override is removed